packages/preview/arkheion/0.1.1/template.typ

```diff
@@ -0,0 +1,146 @@
+#let arkheion(
+  title: "",
+  abstract: none,
+  keywords: (),
+  authors: (),
+  custom_authors: none,
```

Comment:
Can you rename this parameter to `custom-authors`, please? This would be more in line with the rest of the package, and the broader Typst ecosystem.

Reply:
Yes